### DISCUSSION TITLE
How do I set the custom repository for drivers?

### DISCUSSION THREAD

**Discussion by Medvedscak** [webdriverio/webdriverio#11682] [Q&A] [answered]
Hello,

I need to use custom npm repository for drivers in WebdriverIO. It's configured in .npmrc file and all the package.json dependencies are taken from there. But the WebdriverIO is trying to download drivers (i.e. chromedriver) from the Internet (which is restricted in the company).

The latest version of WebdriverIO is used.

I'm trying to figure out if it's possible to configure the source for drivers somehow (couldn't find it in docs) or I have to find more complex workarounds.

Thanks!

**Comment by christian-bromann** [comment, marked as answer]
@Medvedscak it is not possible to define custom download sources but you can download the drivers manually and point WebdriverIO to use them, via e.g.:

```ts
capabilities: [{
  browserName: 'chrome',
  'wdio:chromedriverOptions': { binary: '/path/to/binary' }
}]
```

You can read up on driver options in https://webdriver.io/docs/capabilities#webdriverio-capabilities-to-manage-browser-driver-options

**Reply by Medvedscak** [reply]
Ok, thanks. Not really comfortable of course( Does it use the usual "npmjs" as the source or some own cdn?

**Reply by christian-bromann** [reply]
Browser driver are usually downloaded through a CDN.
